src/dataregistry/db_basic.py

```diff
@@ -173,13 +169,25 @@ def __init__(self, db_connection, get_db_version=True):
         # Load all existing tables
         self._metadata.reflect(self._engine, db_connection.schema)
 
-        # Fetch and save db versioning if present and requested
+        # Fetch and save db versioning, assoc. production schema
+        # if present and requested
+        self._prod_schema = None
         if db_connection.dialect == "sqlite":
             prov_name = "provenance"
         else:
             prov_name = ".".join([self._schema, "provenance"])
+
+            # prov_table = self._metadata.tables[prov_name]
         if prov_name in self._metadata.tables and get_db_version:
             prov_table = self._metadata.tables[prov_name]
+            stmt = select(column("associated_production")).select_from(prov_table)
+            stmt = stmt.order_by(prov_table.c.provenance_id.desc())
+            with self._engine.connect() as conn:
+                results = conn.execute(stmt)
+                conn.commit()
```

Comment:
Don't need `commit` for queries (I think)

Can also remove the commit below if true

Reply:
Yes.  Now removed.